src/openassetio-python/tests/package/pluginSystem/resources/deprecated/deprecatedPlugin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Provides a test PythonPluginSystemPlugin implemented within a single file
+module.
+"""
+
+from openassetio.pluginSystem import PythonPluginSystemManagerPlugin
+
+
+class DeprecatedPlugin(PythonPluginSystemManagerPlugin):
+    # pylint: disable=missing-class-docstring
+
+    @classmethod
+    def identifier(cls):
+        return "org.openassetio.test.pluginSystem.resources.deprecated"
+
+    @classmethod
+    def interface(cls):
```

Comment:
In fact, only `pathB/packagedPlugin/PackagedPlugin.py` needs this `interface` method hack for `test_when_duplicate_identifiers_path_selected_over_entry_point` :disappointed: 

I guess having a pointless `interface` method is consistent with other stub plugins, then. Ideally we'd simplify down these stub plugins to just what's needed for the tests.

Reply:
Hmm. I know i'm normally the consistency guy ... but I did get rid of this because ... reasons.
https://github.com/OpenAssetIO/OpenAssetIO/pull/1312/commits/10421fe9e58bc9322c66200364aeb4b89b980fd4